internal/sre/server.go

```diff
@@ -0,0 +1,1590 @@
+package sre
```

Comment:
Why is this specifically called `sre`. I think we should think about this as a general `httpserver` for `khook` rather than anything specifically for your UI.